composer.json

```diff
@@ -5,7 +5,8 @@
     "license": "GPL-2.0-or-later",
     "require": {
         "dpc-sdp/tide_core": "^1.2.5",
-        "dpc-sdp/tide_media": "^1.2.5"
+        "dpc-sdp/tide_media": "^1.2.5",
+        "drupal/smart_trim": "^1.1"

```

Comment:
We dont need smart_trim here as it's already included in tide core.

Reply:
solved.